Tasks/JavaToolInstallerV0/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "java-tool-installer",
-    "version": "0.211.0",
+    "version": "0.215.0",
```

Comment:
We usually don't update the version here. You can either remove this or update the version in package-lock accordingly